clang/test/Driver/sycl-host-compiler-old-model.cpp

```diff
@@ -76,3 +76,17 @@
 // CHECK-ZC-CPLUSPLUS: "/Zc:__cplusplus"
 // CHECK-ZC-CPLUSPLUS-MINUS: "/Zc:__cplusplus-"
 // CHECK-NO-ZC-CPLUSPLUS-NOT: "/Zc:__cplusplus"
+
+/// -fsycl-host-compiler -save-temps behavior
+// RUN: %clangxx -### -fsycl-host-compiler=g++ -fsycl --no-offload-new-driver \
```

Comment:
What is the behavior in CL mode?

Reply:
`-save-temps` is not supported for `clang-cl`, so there would be an error due to using an unsupported option.